AppWidget/README.md

```diff
@@ -1,332 +1,391 @@
 
-AppWidget Sample
-===================================
+# AppWidget Sample
 
 This sample demonstrates how to use the new Widgets APIs introduced in API level 31.
-The new set of APIs allows an application to build more engaging and beautiful
-widgets.
-The initial set of new APIs focuses on improving existing RemoteView APIs.
+The new set of APIs allows an application to build more engaging and beautiful widgets.
+The initial set of new APIs focuses on improving existing RemoteViews APIs.
```

Comment:
Add backticks on RemoteViews
Also suggest linking it to https://developer.android.com/reference/android/widget/RemoteViews